handler.go

```diff
@@ -51,7 +52,7 @@ func NewGetAllWebhooksHandler(s Service, config HandlerConfig) http.Handler {
 type HandlerConfig struct {
 	V                 Validator
 	DisablePartnerIDs bool
-	GetLogger         GetLoggerFunc
+	GetLogger         sallust.GetLoggerFunc
```

Comment:
This type has been deprecated and removed in the latest sallust release.  If you need a logger function, just declare it inline: `GetLogger func(*context.Context) *zap.Logger`

Reply:
yeah, don't mind anything in `draft`. I have to update them